### PR TITLE
Add CWC2019 Finals Recap & GF Preview

### PR DESCRIPTION
**Pull request by JBHyperion**
+ Newspost recap for Finals Week 1 (inc. twitch VOD embed and results screen image)
+ Grand Final Preview & Scheduling

### CODE REVIEW

File: news/2019-08-01-cwc-2019-preview-grand-final.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+Chile then took a surprising victory of last year's runners-up The United States. Despite losing the opening point, fortune favoured the Chilean squad who repeatedly picked Hidden - a traditional strength of the US - and managed to squeak through by the tiniest of margins, winning each of theif first three picks by less than 35,00 points. The Unites States had avoided HardRock and DoubleTime wherever possible in their previous rounds, and without Hidden to fall back on, their confidence seemed to wane. A last throw of the dice on the HardRock pick [Halozy - Deconstruction Star](https://osu.ppy.sh/beatmapsets/319217#fruits/710613) would have seen them snatch a second game by just 5,000 points, if not for a tragic last-second fail from [-Kurisu-](https://osu.ppy.sh/users/500696) that ensured a 7-1 victory and the chance of another podium placing for last year's 3rd place team.
```
Comment (WalterToro) [inline news/2019-08-01-cwc-2019-preview-grand-final.md]:
```suggestion
Chile then took a surprising victory of last year's runners-up The United States. Despite losing the opening point, fortune favoured the Chilean squad who repeatedly picked Hidden - a traditional strength of the US - and managed to squeak through by the tiniest of margins, winning each of theif first three picks by less than 35,000 points. The United States had avoided HardRock and DoubleTime wherever possible in their previous rounds, and without Hidden to fall back on, their confidence seemed to wane. A last throw of the dice on the HardRock pick [Halozy - Deconstruction Star](https://osu.ppy.sh/beatmapsets/319217#fruits/710613) would have seen them snatch a second game by just 5,000 points, if not for a tragic last-second fail from [-Kurisu-](https://osu.ppy.sh/users/500696) that ensured a 7-1 victory and the chance of another podium placing for last year's 3rd place team.
```

Comment (WalterToro) [reply news/2019-08-01-cwc-2019-preview-grand-final.md]: Yadda yadda, fixed wrong number and a typo here.

Comment (JBHyperion) [reply news/2019-08-01-cwc-2019-preview-grand-final.md]: Fixed things and also changed first "of" to "over"

File: news/2019-08-01-cwc-2019-preview-grand-final.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+date: 2019-08-01 02:00:00 +0000
+---
+
+Five weeks of frantic fruit catching have been and gone, and the field has been whittled down to the final three. From thirty-two teams, only Chile, China and South Korea remain. All have stood on the top step of the podium before, but who will rise to the occasion to claim this year's greatest prize in osu!catch?
+
+![](/wiki/shared/news/banners/CWC_2019_banner.jpg)
+
+A little over a month ago, the curtain was raised on the 2019 edition of the osu!catch World Cup, with over one hundred and eighty competitors showcasing their dashing skills - all in the hopes of becoming the next _osu!catch Champions_. Seventy-six matches, several nail-biting finishes, and countless incredible moments later, and only three teams remain.
+
+If you missed any of the exciting action that led us to this point, VODs from throughout the tournament are available on the [osu!live Twitch channel](https://www.twitch.tv/osulive) for your viewing pleaseure!
```
Comment (Joehuu) [inline news/2019-08-01-cwc-2019-preview-grand-final.md]:
```suggestion
If you missed any of the exciting action that led us to this point, VODs from throughout the tournament are available on the [osu!live Twitch channel](https://www.twitch.tv/osulive) for your viewing pleasure!
```

File: news/2019-08-01-cwc-2019-preview-grand-final.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+Chile then took a surprising victory over last year's runners-up The United States. Despite losing the opening point, fortune favoured the Chilean squad who repeatedly picked Hidden - a traditional strength of the US - and managed to squeak through by the tiniest of margins, winning each of their first three picks by less than 35,000 points. The Unites States had avoided HardRock and DoubleTime wherever possible in their previous rounds, and without Hidden to fall back on, their confidence seemed to wane. A last throw of the dice on the HardRock pick [Halozy - Deconstruction Star](https://osu.ppy.sh/beatmapsets/319217#fruits/710613) would have seen them snatch a second game by just 5,000 points, if not for a tragic last-second fail from [-Kurisu-](https://osu.ppy.sh/users/500696) that ensured a 7-1 victory and the chance of another podium placing for last year's 3rd place team.
```
Comment (Joehuu) [inline news/2019-08-01-cwc-2019-preview-grand-final.md]: `Unites` -> `United`
```suggestion
Chile then took a surprising victory over last year's runners-up The United States. Despite losing the opening point, fortune favoured the Chilean squad who repeatedly picked Hidden - a traditional strength of the US - and managed to squeak through by the tiniest of margins, winning each of their first three picks by less than 35,000 points. The United States had avoided HardRock and DoubleTime wherever possible in their previous rounds, and without Hidden to fall back on, their confidence seemed to wane. A last throw of the dice on the HardRock pick [Halozy - Deconstruction Star](https://osu.ppy.sh/beatmapsets/319217#fruits/710613) would have seen them snatch a second game by just 5,000 points, if not for a tragic last-second fail from [-Kurisu-](https://osu.ppy.sh/users/500696) that ensured a 7-1 victory and the chance of another podium placing for last year's 3rd place team.
```

File: news/2019-08-01-cwc-2019-preview-grand-final.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+![](/wiki/shared/news/2019-08-01-cwc-2019-preview-grand-final/CLvMX-game10-720w.jpg)
+
+An eventual 7-5 victory took them to their second consecutive Loser's Bracket Final next week, once against against China - can they go one (or even two!) better this year? Frankly, if you watch no other match from this year's osu!catch World Cup thus far, make this one a priority - it really was _that_ spectacular.
+
+## Summary
+
+Only two matches remain before this year's osu!catch World Cup winners are decided, and you can catch both of them live in all their glory on the [osu!live Twitch channel](https://www.twitch.tv/osulive) this coming weekend. The Losers' Bracket Final between China and Chile will take place at **16:00 UTC+0 this Saturday, August 3rd**.
+
+The Grand Final will be at **01:00 UTC+0 on Sunday, August 4th** if Chile are victorious in the Losers' Bracket Final, and **12:00 UTC+0 on Sunday** if China come out on top. Be sure to follow [@osugame on Twitter](https://twitter.com/osugame) to receive notifications of all updates, scheduling and live broadcasts!
```
Comment (Joehuu) [inline news/2019-08-01-cwc-2019-preview-grand-final.md]:
```suggestion
The Grand Final will be at **01:00 UTC+0 on Sunday, August 4th** if Chile are victorious in the Losers' Bracket Final, and **12:00 UTC+0 on Sunday** if China comes out on top. Be sure to follow [@osugame on Twitter](https://twitter.com/osugame) to receive notifications of all updates, scheduling and live broadcasts!
```

File: news/2019-08-01-cwc-2019-preview-grand-final.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+If you haven't yet picked up a wicked [profile banner](https://osu.ppy.sh/store/products/608) to show your undying love and support for your favourite teams and players from this year's osu!catch World Cup yet, hurry over to the [osu!store](https://osu.ppy.sh/store/listing) now, as they will shortly become unavailable! All proceeds go towards helping make osu! and World Cups in particular even more awesome, so don't delay!
+
+I hope to see you all cheering our finalists on this weekend for the fantastic conclusion to another year of incredible fruit catching!
```
Comment (Joehuu) [inline news/2019-08-01-cwc-2019-preview-grand-final.md]:
```suggestion
I hope to see you all cheering our finalists this weekend for the fantastic conclusion to another year of incredible fruit catching!
```

File: news/2019-08-01-cwc-2019-preview-grand-final.md (hunk 2 above)
Comment (JBHyperion) [reply news/2019-08-01-cwc-2019-preview-grand-final.md]: typo master strikes again

File: news/2019-08-01-cwc-2019-preview-grand-final.md (hunk 3 above)
Comment (JBHyperion) [reply news/2019-08-01-cwc-2019-preview-grand-final.md]: typo master clone wars

File: news/2019-08-01-cwc-2019-preview-grand-final.md (hunk 4 above)
Comment (JBHyperion) [reply news/2019-08-01-cwc-2019-preview-grand-final.md]: yes

File: news/2019-08-01-cwc-2019-preview-grand-final.md (hunk 5 above)
Comment (JBHyperion) [reply news/2019-08-01-cwc-2019-preview-grand-final.md]: was originally going to say "on Saturday" but then changed my mind, fixed

**Pull request merged**